components/Footer/index.js

```diff
@@ -6,19 +6,73 @@ import React from 'react';
 import {Link} from 'react-router';
 
 class Footer extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {msg: '', email: '', success: ''};
+  }
+
+  onSubmit(e){
+    e.preventDefault();
+    const reg = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    if (!reg.test(this.state.email)){
+      this.setState({msg: 'Please enter a valid email address.'});
+    } else {
+      fetch('http://gettoknow.skookum.com/acton/eform/17086/0005/d-ext-0001', {
+        method: 'post',
+        body: this.state.email,
+      });
+      this.setState({success: 'Thanks for signing up!', email: '', msg: ''});
+    }
+  }
+
+  handleChange(e){
+    this.setState({email: e.target.value});
+  }
+
   render(): ?ReactElement {
+
+    const getYear = new Date().getFullYear();
+
     return (
-      <footer className="Footer">
-        <div className="Footer-links">
-          <Link to="careers" className="Footer-link">Careers</Link>
-          <Link to="open-source" className="Footer-link">Open&nbsp;Source</Link>
-          <a href="https://www.twitter.com/skookum" target="_blank" className="Footer-link">Twitter</a>
-          <a href="https://www.linkedin.com/company/skookum-digital-works" target="_blank" className="Footer-link">Linkedin</a>
-          <a href="https://www.facebook.com/SkookumInc" target="_blank" className="Footer-link">Facebook</a>
-          <a href="https://instagram.com/skookumpeople" target="_blank" className="Footer-link">Instagram</a>
+      <div>
+        <div className="PreFooter">
+          <div className="GetInTouch"><a href="http://skookum.com/contact">Get in touch »</a></div>
```

Comment:
use the Link component here `<Link to="contact">Get in touch</Link>` instead of `<a>` tag